<python>: Python 2.7.15 Darwin compile error

Maintainer: @jan-kardell
Environment: (ath79, archer c7 v2, OpenWrt Master)

Description: python 2.7.15 fails to compile on macos mojave

```
checking whether the C compiler works... no
configure: error: in `/Volumes/openwrt/openwrt/build_dir/hostpkg/Python-2.7.15':
configure: error: C compiler cannot create executables
```

x86_64 also...

HOST_LDFLAGS += \
        $$$$(pkg-config --static --libs libcrypto libssl) -L$(STAGING_DIR_HOST)/lib -Wl$(comma)-rpath-link=$(STAGING_DIR_HOST)/lib

gets U past this, then to:
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... configure: error: in `/home/rossb/OpenWrt/x86_64/r8210/trunk/build_dir/hostpkg/Python-2.7.15':
configure: error: cannot run C compiled programs.
If you meant to cross compile, use `--host'.

checking config.log:
configure:4127: ./conftest
./conftest: error while loading shared libraries: libcrypto.so.1.0.0: cannot open shared object file: No such file or directory

So, now a problem with LD_LIBRARY_PATH, which I am stuck at (so far)

...Bill

--- a/lang/python/python/Makefile
+++ b/lang/python/python/Makefile
@@ -290,11 +290,16 @@ define PyPackage/python/filespec
 endef
 
 HOST_LDFLAGS += \
-	$$$$(pkg-config --static --libs libcrypto libssl) -Wl$(comma)-rpath=$(STAGING_DIR_HOSTPKG)/lib
+	-Wl$(comma)-rpath$(comma)$(STAGING_DIR_HOSTPKG)/lib
 
 ifeq ($(HOST_OS),Linux)
 HOST_LDFLAGS += \
 	-Wl,--no-as-needed -lrt
+endif
+
+ifeq ($(HOST_OS),Darwin)
+HOST_CONFIGURE_VARS += \
+	ac_cv_header_libintl_h=no
 endif
 
 HOST_CONFIGURE_ARGS+= \

--- a/lang/python/python3/Makefile
+++ b/lang/python/python3/Makefile
@@ -285,11 +285,16 @@ define Py3Package/python3/filespec
 endef
 
 HOST_LDFLAGS += \
-	$$$$(pkg-config --static --libs libcrypto libssl) -Wl$(comma)-rpath=$(STAGING_DIR_HOSTPKG)/lib
+	-Wl$(comma)-rpath$(comma)$(STAGING_DIR_HOSTPKG)/lib
 
 ifeq ($(HOST_OS),Linux)
 HOST_LDFLAGS += \
 	-Wl,--no-as-needed -lrt
+endif
+
+ifeq ($(HOST_OS),Darwin)
+HOST_CONFIGURE_VARS += \
+	ac_cv_header_libintl_h=no
 endif
 
 HOST_CONFIGURE_ARGS+= \

